ansible/roles/kong-api/defaults/main.yml

```diff
@@ -1291,319 +1092,71 @@ kong_apis:
     - "{{ statsd_pulgin }}"
     - name: acl
       config.whitelist:
-        - 'appAccess'
-    - name: rate-limiting
-      config.policy: local
-      config.hour: "{{ medium_rate_limit_per_hour }}"
-      config.limit_by: credential
-    - name: request-size-limiting
-      config.allowed_payload_size: "{{ small_request_size_limit }}"
-
-  - name: flagContent
-    uris: "{{ content_prefix }}/v1/flag"
-    upstream_url: "{{ knowledge_mw_service_url }}/v1/content/flag"
-    strip_uri: true
-    plugins:
-    - name: jwt
-    - name: cors
-    - "{{ statsd_pulgin }}"
-    - name: acl
-      config.whitelist:
-        - 'contentAccess'
-    - name: rate-limiting
-      config.policy: local
-      config.hour: "{{ medium_rate_limit_per_hour }}"
-      config.limit_by: credential
-    - name: request-size-limiting
-      config.allowed_payload_size: "{{ small_request_size_limit }}"
-
-  - name: freshDesk
-    uris: "{{ tickets_prefix }}/v1/create"
-    upstream_url: "{{ freshDesk_url }}/api/v2/tickets"
-    strip_uri: true
-    plugins:
-    - name: jwt
-    - name: cors
-    - "{{ statsd_pulgin }}"
-    - name: acl
-      config.whitelist:
-        - 'appAccess'
-    - name: rate-limiting
-      config.policy: local
-      config.hour: "{{ medium_rate_limit_per_hour }}"
-      config.limit_by: credential
-    - name: request-size-limiting
-      config.allowed_payload_size: "{{ small_request_size_limit }}"
-
-  - name: generateDialcodes
-    uris: "{{ dialcode_service_prefix }}/v1/generate"
-    upstream_url: "{{ knowledge_mw_service_url }}/v1/dialcode/generate"
-    strip_uri: true
-    plugins:
-    - name: jwt
-    - name: cors
-    - "{{ statsd_pulgin }}"
-    - name: acl
-      config.whitelist:
-        - 'dialcodeCreate'
-    - name: rate-limiting
-      config.policy: local
-      config.hour: "{{ medium_rate_limit_per_hour }}"
-      config.limit_by: credential
-    - name: request-size-limiting
-      config.allowed_payload_size: "{{ small_request_size_limit }}"
-
-  - name: generateOtp
```

Comment:
CHeck if this API is being used @Hari-stackroute

Reply:
@SMYALTAMASH This API is not removed, its showing in the committed file in a different line

Reply:
Might be because of lot of deletion

Reply:
revert this back @amit-tarento